src/clusterfuzz/_internal/cron/cleanup.py

```diff
@@ -761,7 +761,7 @@ def notify_closed_issue_if_testcase_is_open(policy, testcase, issue):
   if issue.status in [policy.status('fixed'), policy.status('verified')]:
     issue_comment = (
         f'ClusterFuzz testcase {testcase.key.id()} is still reproducing '
-        f'on the latest available build{crash_revision}.'
+        f'on the latest available build {crash_revision}.'
```

Comment:
Btw this was intentional, not to leave a space between `build` and `.` when crash revision is empty.

Reply:
space added on line 760